src/vstest.console/Resources/Resources.resx

```diff
@@ -807,4 +807,7 @@
   <data name="InvalidTestSessionCorrelationId" xml:space="preserve">
     <value>Invalid testSessionCorrelationId</value>
   </data>
+  <data name="WarningEmulatedOnArm64" xml:space="preserve">
+    <value>For better performance, please consider running the native arm64 runner, vstest.console.arm64.exe</value>
```

Comment:
I posted a comment internally. This should probably include slightly more info about why users is seeing the warning. So they don't have to connect the dots that they are running as x86/x64 process.

Reply:
will do